keras_cv/models/segmentation/segment_anything/sam_mask_decoder.py

```diff
@@ -0,0 +1,230 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from keras_cv.api_export import keras_cv_export
+from keras_cv.backend import keras
+from keras_cv.backend import ops
+from keras_cv.layers.serializable_sequential import SerializableSequential
+from keras_cv.models.segmentation.segment_anything.sam_layers import MLP
+
+
+@keras_cv_export("keras_cv.models.MaskDecoder")
+class MaskDecoder(keras.models.Model):
```

Comment:
As mentioned before, to avoid confusion, probably best if this is called SAMMaskDecoder or something along those lines

Reply:
Done.